client/index.jsx

```diff
@@ -1,39 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import $ from 'jquery';
-import Calendar from './components/Calendar.jsx';
-const cdrGen = require('./cdr.js');
-
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      months: this.props.months
-    }
-  }
-
-  render () {
-    return (
-    <div>
-      <h2>Availability</h2>
-      <div>
-        <Calendar months = {this.state.months}/>
-      </div>
-      <iframe>FRAME</iframe>
-      <button> select a date
-      </button>
-    </div>
-    )
-  }
+import App from './components/App.jsx';
+
+if (window.location.href.split('/').slice(-1)[0]) {
```

Comment:
I know this works (because you showed us), so it's ok as-is.  When you asked for me to find a simpler way that works, I came up with this for a default experience id of 1:
const URL = window.location.href.split('/');
const expId = URL[URL.length - 1] || 1;
you could probably combine them if you wanted to make it even more compact, but it might make it harder to read